storybook/main.cpp

```diff
@@ -41,6 +41,13 @@ int main(int argc, char *argv[])
     QGuiApplication::setOrganizationDomain(QStringLiteral("status.im"));
     QGuiApplication::setApplicationName(QStringLiteral("Status Desktop Storybook"));
 
+    QCommandLineParser cmdParser;
+    cmdParser.addHelpOption();
+    cmdParser.addPositionalArgument(QStringLiteral("page"), QStringLiteral("Open the given page on startup"));
+    if (!cmdParser.parse(app.arguments())) {
+        qWarning() << "Error (ignored) while parsing cmd line arguments:" << cmdParser.errorText();
```

Comment:
```suggestion
        qWarning() << "Warning (ignored) while parsing cmd line arguments:" << cmdParser.errorText();
```

Reply:
Well, it technically _is_ an error, we just ignore it :)